tests/loginInvalidUser.test.js

```diff
@@ -0,0 +1,20 @@
+const { test, expect } = require('@playwright/test');
+const { LoginPage } = require('../pages/loginPage.js');
+import { generateRandomString } from '../utils/random.js';
+
+const randomUser = 'invalidUser' + generateRandomString(4);
+const randomPassword = generateRandomString(10);
+
+test.describe('Redmine login', () => {
+    test('should not login with random user and password', async ({ page }) => {
+        const loginPage = new LoginPage(page);
+        await loginPage.goto('https://www.redmine.org/login');
+        const user = {
+            username: randomUser,
+            password: randomPassword,
+        };
+        await loginPage.loginUser(user);
+        const flashError = await loginPage.getFlashError();
```

Comment:
@karenn32 For the test cases, where we must check, for example, the login process directly - no need to create a separated function for that, it makes sence, when the same process will be executed in the 'before' hooks of all other test specs, but here - only cause the readability issues. Also, that will be good to go when the process and script for it is complex (check sorting functionality of the table's columns fro example), in such case - create functions with readable JSDoc - great idea.